plugins/inputs/fritzbox/fritzbox_test.go

```diff
@@ -11,85 +11,138 @@ import (
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
 	"github.com/influxdata/telegraf/plugins/parsers/influx"
 	"github.com/influxdata/telegraf/testutil"
 )
 
-func TestConfig(t *testing.T) {
+func TestValidDefaultConfig(t *testing.T) {
```

Comment:
IMHO, testing valid default config would include using `testutil.DefaultSampleConfig` to actually test if the given default config would include correct sample property names and values..

Reply:
Applying `testutil.DefaultSampleConfig` would require to adapt the TLS section, as the commented values are not suitable defaults. I actually have reused the TLS section from another plugin's sample.conf.
How is the TLS section normally handled? Should I set the commented TLS fields to empty, which are the expected defaults unless user provides concrete certificate files?

Reply:
Let's keep it as-is for now...